tee-worker/omni-executor/intent/executors/cross-chain/src/cross_chain_swap_tests.rs

```diff
@@ -102,7 +102,7 @@ async fn simple_cross_chain_swap() {
 			mockall::predicate::eq(pumpx_wallet_index),
 			mockall::predicate::eq(pumpx_wallet_omni_account),
 		)
-		.times(1)
+		.times(2)
```

Comment:
This PR introduced one redundant call.

Reply:
True - I'll improve it further in the upcoming PRs where I put the common value parsing ahead